.github/workflows/release.yml

```diff
@@ -16,44 +17,38 @@ jobs:
         with:
           token: ${{ secrets.GH_TOKEN }}
 
-      - name: Use Node.js 20.17.0
+      - name: Set git user to carbon-bot
+        run: |
+          git config --global user.email "carbon@us.ibm.com"
+          git config --global user.name "carbon-bot"
+
+      - name: Use Node.js 20.x LTS
         uses: actions/setup-node@v4
         with:
-          node-version: '20.17.0'
+          node-version: '20.x'
+          cache: 'yarn'
+          registry-url: 'https://registry.npmjs.org/'
+          always-auth: true
+          auth-token: ${{ secrets.NPM_TOKEN }}
 
       - name: Install dependencies 🔧
         run: yarn install --immutable --check-cache
 
-      - name: Set git user and get git into the right state
-        run: |
-          git config --global user.email "carbon@us.ibm.com"
-          git config --global user.name "carbon-bot"
-          git checkout master
+      - name: Generate search index for docs website
+        run: lerna run build:search --scope @carbon/charts-docs # should be committed in next step
 
-      - name: Create version and change logs
+      - name: Create version and change logs and commit to master
```

Comment:
As this commits the new version numbers to master, it can also commit the rebuilt search index for the docs website.